qa/pull-tester/rpc-tests.py

```diff
@@ -143,7 +143,7 @@ def option_passed(option_without_dashes):
 bad_opt_str="Unrecognized option: %s"
 for o in opts | double_opts:
     if o.startswith('--'):
-        if o not in framework_opts + test_script_opts + private_double_opts:
+        if o.split("=")[0] not in framework_opts + test_script_opts + private_double_opts:
```

Comment:
this change is unrelated.

Reply:
true, I found this bug while working on this PR, and the fix is so small and simple...